src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -64,26 +67,23 @@ const AppCard: React.FC<AppCardProps> = ({
 
       setSubscribing(true)
       try {
-        await notifyClientProxy?.subscribe({
-          account: userPubkey,
-          appDomain: new URL(url).host
-        })
+        await subscribe(undefined, host)
       } catch (error) {
         logError(error)
         setSubscribing(false)
         showErrorMessageToast(`Failed to subscribe to ${name}`)
       }
     },
-    [userPubkey, name, description, logo, url, setSubscribing, subscribed, notifyClientProxy]
+    [userPubkey, name, description, logo, url, setSubscribing, subscribed]
   )
 
   const handleNavigateApp = () => {
     if (subscribed) {
       try {
         const appDomain = new URL(url).host
-        const topic = activeSubscriptions.find(sub => sub.metadata.appDomain === appDomain)?.topic
+        const topic = activeSubscriptions?.find(sub => sub.metadata.appDomain === appDomain)?.topic
         if (topic) {
-          nav(`/notifications/${topic}`)
+          nav(`/notifications/${appDomain}`)
```

Comment:
Let's use the `NAVIGATION` object for any href calls. We have open PR for those other hrefs but I guess I need to update that PR after these changes.

-  https://github.com/WalletConnect/web3inbox/pull/476